Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            PersonalInfo();
+            PersonalInfo();
+        }
+
+        public static void PersonalInfo(){
```

Comment:
A function should be named after some action because they do something. So a verb.
``PromptPersonalInfo`` sounds better

Reply:
Done!